test/functional/eval/timer_spec.lua

```diff
@@ -96,6 +96,7 @@ describe('timers', function()
     source([[
       func! AddItem(timer)
         call nvim_buf_set_lines(0, 2, 2, v:true, ['ITEM 3'])
+        call getchar(1)
```

Comment:
I added this to match `FeedAndPeek()` in `test_timers.vim`. I don't know what to do with the second timer that calls `Interrupt()`.